frontend/mockData/output.ts

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
maybe get rid of this if not needed

Reply:
[Will be removed](https://alanyan.atlassian.net/browse/SG-56), thanks